feat(2624): add design and modify flow in MASE

lengyel-arpad85 opened the pull request:
<!--- Pull request titles should follow conventional commit format. -->

<!--
Thanks for opening a PR! Your contribution is much appreciated.
In order to make sure your PR is handled as smoothly as possible we request that you follow the checklist sections below.
Choose the right checklist for the change that you're making:
-->

## Changes proposed in this pull request
<!--
Provide a succinct description of what this pull request entails.
-->
solves most of #2624 

- removed / renamed `shoe-shop` page/url to `mock-idp/consent` for it to make more sense 
- added functionality to create a new account
- added edit account functionality & a way to add funds to an account
- added consistent design to pages (see screenshots)
- made modifications to the consent pages, in structure and design

| Screenshots (v2) |   |   |
|--------|--------|--------|
| ![](https://github.com/interledger/rafiki/assets/165793743/0b471e28-363a-4704-8a59-60833cb65885) | ![](https://github.com/interledger/rafiki/assets/165793743/15fc5a5f-eab6-4bf1-8f3b-c32faadc668b) | ![](https://github.com/interledger/rafiki/assets/165793743/67c3be32-aecb-48b9-be7f-8911d2c78591) |

_____________

| Screenshots (old) |   |   |
|--------|--------|--------|
| ![](https://github.com/interledger/rafiki/assets/165793743/8d785df4-2a63-4016-bc47-67967e59a97c) | ![](https://github.com/interledger/rafiki/assets/165793743/ea337b9d-d5d9-4be4-b771-b2076a8aed6d) | ![](https://github.com/interledger/rafiki/assets/165793743/9b0bf936-0ff9-48f0-9a01-57398d8456be) |
| ![](https://github.com/interledger/rafiki/assets/165793743/d2713b82-49cd-48b3-8fab-08d645548314) | ![](https://github.com/interledger/rafiki/assets/165793743/b0ed69d1-b4a1-4a9c-85b7-dd312390c624) | ![](https://github.com/interledger/rafiki/assets/165793743/8bccb5fe-ec9b-4c77-ad18-89e7814bf982) | 


## Context
<!--
What were you trying to do?
Provide further details about how the feature should be tested/reviewed if necessary.
Link issues here -  using `fixes #number`
-->

## Checklist
<!--
Checklist items become clickable check boxes once the pull request is created. There is no need to edit them now.
-->

- [x] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [x] Make sure that all checks pass
- [ ] Bruno collection updated

Tymmmy commented on localenv/mock-account-servicing-entity/app/components/Button.tsx:
```diff
@@ -0,0 +1,46 @@
+          'disabled:bg-red-200 bg-red-500 hover:bg-red-600 shadow-md text-white'
+      },
+      size: {
+        sm: 'px-2 py-1 rounded-md font-medium',
```
you could add rounded and font size to general styles, as they are the same for both sizes

mkurapov commented:
A few of us took a look at the screenshots, and had some comments:

In general, looks much better, thank you! 
The main thing we wanted to do was to distinguish it a bit more from the Admin UI, just to avoid any confusion for people working through the local playground, between what would be part of Rafiki, or something external.

Some thoughts we had:
- Adding the name of the instance (eg "Happy Life Bank" or "Cloud Nine Wallet") instead of "Mock ASE" , and adding some kind of icon to go along with it, either the wallet icon (like the one in the new test wallet/rafiki.money redesign) or the bank icon
- Adding something like global disclaimer to the footer of the page, explaining that this is just a "Mock ASE" and not for use as a production interface
- Mainly to differentiate from the Rafiki UI, there was an idea was to move the two tabs to the top of the page (at least somewhere other than the left side bar)
- And in general, some minor changes in the color, to contribute to the separation between the mock ASE and Admin UI

I'll start a thread in slack in case someone else had additional thoughts.

JoblersTune commented:
How about this for a different look:
![image](https://github.com/interledger/rafiki/assets/77615158/111a0e8d-c6ec-4411-9484-6d8a97f3137a)
We could have a different background for the wallet ASE using it's wallet svg instead of the banks.

JoblersTune commented:
Attached are the assets I created:
file:///home/sarah/Downloads/wallet-white.svg
file:///home/sarah/Downloads/wallet-background.png
file:///home/sarah/Downloads/background.png
file:///home/sarah/Downloads/Bank%20Icon%20(1).svg



Colors:

- EBF8FF
- 3533A0
- 242374

lengyel-arpad85 commented:
> A few of us took a look at the screenshots, and had some comments:
> 
> In general, looks much better, thank you! The main thing we wanted to do was to distinguish it a bit more from the Admin UI, just to avoid any confusion for people working through the local playground, between what would be part of Rafiki, or something external.
> 
> Some thoughts we had:
> 
> * Adding the name of the instance (eg "Happy Life Bank" or "Cloud Nine Wallet") instead of "Mock ASE" , and adding some kind of icon to go along with it, either the wallet icon (like the one in the new test wallet/rafiki.money redesign) or the bank icon
> * Adding something like global disclaimer to the footer of the page, explaining that this is just a "Mock ASE" and not for use as a production interface
> * Mainly to differentiate from the Rafiki UI, there was an idea was to move the two tabs to the top of the page (at least somewhere other than the left side bar)
> * And in general, some minor changes in the color, to contribute to the separation between the mock ASE and Admin UI
> 
> I'll start a thread in slack in case someone else had additional thoughts.

The update:
 
- changed left menubar to top menu
- added the separate logos and background image for each instance
- added instance name to meta title and topmenu
- added infobar to bottom of pages with the disclaimer
- changed colors used (see new screenshots)

lengyel-arpad85 commented on localenv/mock-account-servicing-entity/app/routes/rates._index.tsx:
```diff
@@ -0,0 +1,61 @@
+  return json({ currencies, rates })
+}
+
+export default function Rates() {
```
I've changed the route for the page to rates/view, so the original route is now accessible by the backend services again

lengyel-arpad85 commented on localenv/mock-account-servicing-entity/app/routes/accounts.$accountId.tsx:
```diff
@@ -1,53 +1,329 @@
+                    placeholder='0'
+                    min={1}
+                    error={response?.errors.addLiquidity.fieldErrors.amount}
+                    onChange={onAmountChange}
+                  />
```
zod coerce to bigint does not work for decimal numbers so I've changed that, it now accepts values containing decimals
